packages/site/cms-site/src/iframebridge/IFrameBridge.module.scss

```diff
@@ -0,0 +1,5 @@
+.childrenWrapper {
+    position: relative;
+    z-index: 1;
+}
+
```

Comment:
Shouldn't this empty line be prevented by eslint/prettier? 
Is there maybe an issue with linting `scss` files?

Reply:
Good catch. I configured prettier to also check css and scss files: d7bf1257f9cb18e1c10a4b63bba21238be9b044c